development/aws-resources.py

```diff
@@ -0,0 +1,46 @@
+"""
```

Comment:
@kai-nguyen-aligent there's a fair bit in this PR that doesn't seem related to the PR topic/description.

Reply:
@tvhees So sorry. I think it's my bad when I create this branch. Look like it include the changes from the other PR. Let me create another one with only the changes for node18 & esm